tests/e2e/lib/apps.go

```diff
@@ -521,7 +521,7 @@ func VerifyBackupRestoreData(artifact_dir string, namespace string, routeName st
 	}
 
 	//Verifying backup-restore data only for CSI as of now.
-	if backupRestoretype == CSI || backupRestoretype == CSIDataMover || backupRestoretype == RESTIC {
+	if backupRestoretype == CSI || backupRestoretype == CSIDataMover || backupRestoretype == RESTIC || backupRestoretype == KOPIA {
```

Comment:
Not direct related to the PR, but I would remove the comment "Verifying backup-restore data only for CSI as of now" (it does not make sense to me) and remove the if, as it has all possible backupRestoretype

Reply:
NIT, should remove in another PR as currently PROW is having hard time and don't want to have all the jobs restarted.

Reply:
I will leave backupRestoretype here as it may be some null etc.. so prefer to be on a safe side

Reply:
Removed the comment and updated as well the dpa_deployment_suite_test.go to address #1176. Still some restic in logs is left, but only in the places where restic is directly consumed.